src/QMCWaveFunctions/EinsplineSetBuilderOld.cpp

```diff
@@ -275,7 +275,7 @@ EinsplineSetBuilder::MuffinTinPath(int ti, int bi, int tin)
   return groupPath.str();
 }
 
-#ifdef QMC_CUDA
```

Comment:
What is the logic behind keeping this commented out code?

@ye-luo  Keep or delete?

Reply:
A decision is needed to fully remove all the unused routines in the builder. The builder has the old I/O code for APW(MuffinTin). If we delete, better to purge it everywhere not just the builder in one short.

Reply:
OK. Perhaps do this when the new APW work is published and productionized.